spec

```diff
@@ -691,7 +691,7 @@ beginseg
     include "build/data/code/code_801A7B10.bss.o"
     include "build/src/code/code_801AA020.o"
     include "build/data/code/code_801AA020.bss.o"
-    include "build/src/code/z_game_over.o"
+    include_readonly "build/src/code/z_game_over.o"
```

Comment:
What's this for?

Reply:
There's a very rare (assumed) bug where read-write data somehow ends up in rodata. It's believed this is a linker issue and since we (thankfully) don't use the original linker, the `include_readonly` directive was added as a hack to place the .data section next to the .rodata section in the linker script.

Reply:
Oh, weird.